charts/argo-workflows/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v2
+name: argo-workflows
+description: A Helm chart for Argo Workflows
+type: application
+version: 0.1.0
+icon: https://raw.githubusercontent.com/argoproj/argo/master/docs/assets/argo.png
```

Comment:
This link should be updated to https://raw.githubusercontent.com/argoproj/argo-workflows/master/docs/assets/argo.png